ui/pages/send/send-content/send-asset-row/send-asset-row.component.js

```diff
@@ -53,7 +54,10 @@ export default class SendAssetRow extends Component {
             assetSelected: token ? ERC20 : this.props.nativeCurrency,
           },
         });
-        this.props.setSendToken(token);
+        this.props.updateSendAsset({
```

Comment:
Similar concern here re: the extra delay, given the new balance update call.

Reply:
fixed in https://github.com/MetaMask/metamask-extension/pull/10965/commits/f94694c0f55a9ccdfe96a7563f4acb4acb56d70b